tests/api/conftest.py

```diff
@@ -1,8 +1,9 @@
 """Middle level conftest visible for tests directory"""
+import random
 import pytest
 import requests
-from faker import Faker
 from tests.api.constants import URL
+from tests.api.__init__ import *
```

Comment:
__init__ method is not supposed to be imported. It indexes directory as Python package. I'd explain more details in further. 

Now, you can just import tests.api 

2. Pls avoid wildcard (*) imports. It can cause a side effect with name conflicts, etc ..